test_node_env/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "test_node_env",
+  "version": "1.0.0",
+  "description": "A module helps to create testing enviroment",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
```

Comment:
best to put the author name here, see the outer `package.json`.

Reply:
Okee